awslimitchecker/checker.py

```diff
@@ -218,6 +218,19 @@ def get_project_url(self):
         """
         return self.vinfo.url
 
+    def remove_skipped_services(self, service_to_skip=None):
+        """
+        If ``service_to_skip`` is specified, the listed services
+        are removed from the services[] class list.
+
+        :param service_to_skip: the name(s) of one or more services to avoid
+          returning limits for
+        :type service_to_skip: list
+        """
+        if service_to_skip is not None:
```

Comment:
The code and signature of this function would be much simpler of ``service_to_skip`` defaulted to an empty list instead of None. That would allow removal of the ``if`` branch here, and make the signature of the function look more like what it does. Also, ``service`` in the variable name should be pluralized.